src/test/java/seedu/guest/logic/parser/AddCommandParserTest.java

```diff
@@ -91,81 +103,107 @@ public void parse_compulsoryFieldMissing_failure() {
         String expectedMessage = String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddCommand.MESSAGE_USAGE);
 
         // missing name prefix
-        assertParseFailure(parser, VALID_NAME_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB + DATE_RANGE_DESC_BOB
+        assertParseFailure(parser, VALID_NAME_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB
+                        + ROOM_DESC_BOB + DATE_RANGE_DESC_BOB
```

Comment:
Should the indentation of the line follow that of the assertParseSuccess above?

Reply:
Rectified!